packages/dashboard-frontend/src/services/bootstrap/namespaceProvisionWarnings.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2018-2023 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { container } from '@/inversify.config';
+import { getKubernetesNamespace } from '@/services/backend-client/kubernetesNamespaceApi';
+import { WarningsReporterService } from '@/services/bootstrap/warningsReporter';
+import { AppState } from '@/store';
+import { selectAdvancedAuthorization, selectAutoProvision } from '@/store/ServerConfig/selectors';
+
+const warningsReporterService = container.get(WarningsReporterService);
+
+export async function checkNamespaceProvisionWarnings(getState: () => AppState): Promise<void> {
+  let username: string | undefined = undefined;
+  const state = getState();
+  const autoProvision = selectAutoProvision(state);
+  if (autoProvision === false) {
+    if (username === undefined) {
+      username = await getUsername();
+    }
+    warningsReporterService.registerWarning(
+      'autoProvisionWarning',
+      `Automatic namespace provisioning is disabled. Namespace for ${username} might not have been configured yet. Please, contact the administrator.`,
+    );
+  }
+
+  const advancedAuthorization = selectAdvancedAuthorization(state);
+  if (advancedAuthorization === undefined || Object.keys(advancedAuthorization).length === 0) {
```

Comment:
1. Explicitly check if user is in `denyUsers`. If so, print the warning `Access for <username> is forbidden. Please contact the administrator`
2. If AdvancedAuthorization is enabled (advancedAuthorization.allowGroups || advancedAuthorization.denyGroups || advancedAuthorization.allowUsers || advancedAuthorization.denyUsers) print the warning `Advanced authorization is enabled. User <username> might not be allowed. Please, contact the administrator.`

Reply:
Thanks. I have fixed it.

Reply:
@tolusha We had a meeting with @ibuziuk about the ability to get a username from Dashboard until namespace provision and decided to rewrite warnings without the username.